tradingstrategy/testing/mock_client.py

```diff
@@ -41,6 +37,35 @@ def fetch_candle_dataset(self, bucket: TimeBucket):
 
     def fetch_all_liquidity_samples(self, bucket: TimeBucket) -> Table:
         raise MockClientNotImplemented()
+    
+    def get_default_quote_token_address(self, factory_address: str | None = None) -> str:
```

Comment:
I don't think there is "default" quote token because it can be either USDC or USDT.

Maybe just pass any inputs needed as an argument for the constructor?